introduction.md

```diff
@@ -1 +1,3 @@
-I introduce myself :) I liked HTML & CSS 
\ No newline at end of file
+I introduce myself :) I liked 
+*HTML & CSS
+*Ruby 
\ No newline at end of file
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?